Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs

```diff
@@ -88,6 +92,17 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateImage();
 		}
 
+		void SetControlPropertiesFromProxy()
+		{
+			foreach (UIControlState uiControlState in s_controlStates)
+			{
+				Control.SetTitleColor(UIButton.Appearance.TitleColor(uiControlState), uiControlState); // if new values are null, old values are preserved.
+				Control.SetTitleShadowColor(UIButton.Appearance.TitleShadowColor(uiControlState), uiControlState);
+				Control.SetBackgroundImage(UIButton.Appearance.BackgroundImageForState(uiControlState), uiControlState);
+				Control.SetImage(UIButton.Appearance.ImageForState(uiControlState), uiControlState); // UpdateImage() renders this useless.
```

Comment:
Go ahead and take out the SetImage line for now; I have an idea of how we can handle that, but we can create another PR for that.

The BackgroundImage test is crashing when I run it in the simulator - it looks like you're referencing `image.png` in the custom renderer, but that image doesn't seem to be in the project.